src/scripts/main.js

```diff
@@ -1,3 +1,7 @@
 'use strict';
 
-// write code here
+const field = document.querySelector('.wall');
+const spider = document.querySelector('.spider');
```

Comment:
it's a good practice to add suffix ref/el to the variable with a reference to node element
```suggestion
const fieldRef = document.querySelector('.wall');
const spiderRef = document.querySelector('.spider');
```